docs/rdd-programming-guide.md

```diff
@@ -105,6 +105,7 @@ Spark {{site.SPARK_VERSION}} works with Python 3.6+. It can use the standard CPy
 so C libraries like NumPy can be used. It also works with PyPy 2.3+.
 
 Python 2, 3.4 and 3.5 supports were removed in Spark 3.1.0.
+Python 3.6 support was removed in Spark 3.2.0.
```

Comment:
Python 3.6 support is not removed in 3.2 yet but just deprecated.

Reply:
Then shall we make the PR after Python 3.6 is deprecated?

Reply:
As #33139 will be merged soon anyway, we can just go ahead.
The wording is not correct because we don't remove Python 3.6 support yet.

Reply:
Makes sense!